x-pack/plugins/security_solution/common/api/entity_analytics/common/common.schema.yaml

```diff
@@ -23,20 +23,18 @@ components:
               properties:
                 cluster:
                   type: object
-                  properties:
-                    manage_index_templates:
-                      type: boolean
-                    manage_transform:
-                      type: boolean
+                  additionalProperties:
```

Comment:
this is outside the scope of this ticket but it's weird to me that we're using OpenAPI to type the privileges object, when we already have TS types for it on kibana side, right?

Reply:
Yeah, but from what I know, we can't reuse TS types inside OpenAPi schemas. 😞